book/src/tutorial/preparation.md

```diff
@@ -0,0 +1,67 @@
+# Preparation
+In order to install gir and nicely structure the project, there are a few things to set up.
+
+## Set up the project folder
+In order to keep the project folder nicely organized, lets create a folder where we will work in and initialize the repo. We will create two library crates. pango will contain the safe wrapper crate and because it is a wrapper for the unsafe bindings, we create the pango-sys crate within the pango crate. If no Cargo.toml file is present in the sys create, a new one will be generated, so lets be safe and delete the automatically created file before we begin. The following commands will set up the project folder as described. 
+```sh
+mkdir gir-tutorial
+cd gir-tutorial/
+git init
+cargo new pango --lib
+cd pango
+cargo new pango-sys --lib
+rm pango-sys/Cargo.toml
+```
+We will also create a file called "Gir.toml" in each of the crates.
+```sh
+touch Gir.toml
+touch pango-sys/Gir.toml
+cd ..
+```
+
+## Installing gir
+Of course we also need to download and install [gir].
+```sh
+git submodule add https://github.com/gtk-rs/gir
```

Comment:
I would make the submodule don't auto update by default, see https://github.com/gtk-rs/gtk4-rs/blob/master/.gitmodules#L4, otherwise anyone using your library from git will have the useless submodules checked out

Reply:
Fixed in a96242c7afab139df04823ecaf180d839fda12f2